templates/expo-template-default/app/+html.tsx

```diff
@@ -0,0 +1,28 @@
+import { ScrollViewStyleReset } from 'expo-router/html';
+import { type PropsWithChildren } from 'react';
+
+// This file is web-only and used to configure the root HTML for every
+// web page during static rendering.
+// The contents of this function only run in Node.js environments and
+// do not have access to the DOM or browser APIs.
+export default function Root({ children }: PropsWithChildren) {
```

Comment:
I'm not sure about this one, maybe @EvanBacon can weigh in whether we should include this file on the default template.

I think we should update the web side so that the website has a header nav instead of bottom tabs, and it would look more at home on the web (beyond the scope of this PR), so this is more of a plaster.

Reply:
My idea here was just to point out that this special file can be used, not making the page look visually better - that can certainly be done too.

Reply:
We just removed this file from the template recently in favor of `npx expo customize`.